notifications/index.js

```diff
@@ -0,0 +1,116 @@
+import { NOTIFICATION_TYPE } from '../server/helpers/constants';
+
+const NOTIFICATIONS = {
+  ACTIVATE_YOUR_ACCOUNT: {
+    description: 'Complete your registration by verifying your email',
+    type: NOTIFICATION_TYPE.INFO,
+    url: '/',
+  },
+  BAN_USER: {
+    description: 'Your account has been locked for security reasons.',
+    type: NOTIFICATION_TYPE.DANGER,
+    url: '/',
+  },
+  CANCEL_VISIT: {
+    description: 'Visitation to property cancelled',
```

Comment:
We need to add the property name here.

```suggestion
    description: 'The visitation to [property] property has been cancelled',
```